.github/workflows/tests.yml

```diff
@@ -250,6 +252,15 @@ jobs:
             ./build/examples/characteristics_advection
           ;;
           esac
+
+          cmake --install build --prefix $DDC_ROOT
+          rm -rf build
```

Comment:
Not clear why it is useful. Perhaps you can use different names for each build phase?

Reply:
What is not clear ? Removing the `build` directory ? I think it was easier to copy-paste this than renaming all each time